### ISSUE TITLE
Remove prop alt from button component

### ISSUE THREAD

**Issue by Veyfeyken** [digipolisantwerp/acpaas-ui_react#202]
A button element cannot contain an alt-attribute, it only applies to images.

```html
<button class="a-button " title="" alt="" aria-label="">Primary</button>
```
